db/migrate/20210706093159_add_status_to_inventories.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToInventories < ActiveRecord::Migration[6.1]
+  def change
+    add_column :inventories, :status, :integer
```

Comment:
try to add it into db/migrate/20210706042737_create_inventories.rb for easy checking